src/hotspot/share/jfr/jfr.hpp

```diff
@@ -54,6 +59,10 @@ class Jfr : AllStatic {
   static void exclude_thread(Thread* thread);
   static bool is_excluded(Thread* thread);
   static void include_thread(Thread* thread);
+
+#ifdef JFR_HAVE_INTRINSICS
```

Comment:
can be declared unconditionally to #ifdef JFR_HAVE_INTRINSICS (call site will have the conditional).

Reply:
fixed.